Tests/Storage/DataManagerTest.php

```diff
@@ -89,6 +97,10 @@ public function testSaveLoadDrop() {
 		// ensure only data of the correct flow has been dropped
 		$this->assertEquals(array(), $this->dataManager->load($createLocationFlow));
 		$this->assertEquals($this->createVehicleFlowData, $this->dataManager->load($createVehicleFlow));
+
+		// Test one flow exists, the other doesn't
```

Comment:
better: ensure only data of the correct flow exists in the storage
And this block should be moved up before the assertions for `load`.